src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/UpdateOrgContactInformationIntegrationTest.java

```diff
@@ -0,0 +1,113 @@
+package uk.gov.hmcts.reform.professionalapi;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.professionalapi.controller.request.ContactInformationCreationRequest;
+import uk.gov.hmcts.reform.professionalapi.util.AuthorizationEnabledIntegrationTest;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.professionalapi.helper.OrganisationFixtures.createContactInformationRequest;
+import static uk.gov.hmcts.reform.professionalapi.helper.OrganisationFixtures.organisationRequestWithAllFieldsAreUpdated;
+import static uk.gov.hmcts.reform.professionalapi.helper.OrganisationFixtures.organisationRequestWithMultipleAddressAllFields;
+
+class UpdateOrgContactInformationIntegrationTest extends AuthorizationEnabledIntegrationTest {
+
+    @Test
+    void update_addressline1_for_organisation_with_prd_admin_role_should_return_200() {
+        ContactInformationCreationRequest contactInformationCreationRequest =
+            createContactInformationRequest().build();
+
+        Map<String, Object> updateResponse = professionalReferenceDataClient
+            .updateOrgContactInformation(contactInformationCreationRequest,
+                hmctsAdmin,createOrganisationRequest(),true);
+
+        assertThat(updateResponse).containsEntry("http_status", 200);
```

Comment:
add assertions to verify updated address details

Reply:
added asserts

Reply:
you updated contact information with below details, but assertions verifying different data and dx address verification is missing. Add prefix "updated" to each new field so that it is easy to identify what is updated data.

.addressLine1("addLine1")
                .addressLine2("addLine2")
                .addressLine3("addLine3")
                .country("country")
                .county("county")
                .townCity("town-city")
                .uprn("uprn1")
                .postCode("post-code")
                .dxAddress(Arrays.asList(dxAddressCreationRequest()
                    .dxNumber("DX 1234567890")
                    .dxExchange("dxExchange").build()));